### DISCUSSION TITLE
Unable to update DockerHub Readme

### DISCUSSION THREAD

**Discussion by buchdag** [nginx-proxy/nginx-proxy#1868] [Q&A] [answered]
It seems like I can't update the Readme on DockerHub anymore, it keep being reset to ![latest 0.8.0](https://img.shields.io/badge/latest-0.8.0-green.svg?style=flat) ![nginx 1.21.4](https://img.shields.io/badge/nginx-1.21.4-brightgreen.svg) whenever I hit the `Update` button and refresh. Does anyone have any idea on how to solve this ? There isn't even a support you can contact on Dockerhub if you're not on a paid plan.

**Comment by tkw1536** [comment]
No idea, but maybe someone in https://forums.docker.com/ can help? That seems accessible without a subscription.

**Comment by buchdag** [comment, marked as answer]
Seems to be fixed.
